src/main/java/com/epam/repository/PersonDAO.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.repository;
+
+import com.epam.model.Person;
+import com.epam.model.PersonRoleEnum;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+
+import java.sql.ResultSet;
+import java.util.List;
+
+@Repository
+public class PersonDAO {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public PersonDAO(NamedParameterJdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public List<Person> getPersons(){
+        return jdbcTemplate.query("SELECT * FROM person",
+                (ResultSet rs, int rowNum) -> Person.builder()
```

Comment:
please, remove ResultSet and int, and use lambda type method reference here (see discussion of prev request)

Reply:
Done